### PR TITLE
VZ-7041: Validate the view-user role for verrazzano user

### PR DESCRIPTION
**Pull request by bhatpmk**
Adds a new test case to validate the verrazzano user has view-user client role

### CODE REVIEW

File: platform-operator/controllers/verrazzano/component/keycloak/keycloak.go [hunk 1]
```diff
@@ -1562,12 +1562,12 @@ func updateRancherClientSecretForKeycloakAuthConfig(ctx spi.ComponentContext) er
 func addClientRoleToUser(ctx spi.ComponentContext, cfg *restclient.Config, cli kubernetes.Interface, userName, clientID, targetRealm, roleName string) error {
 	kcPod := keycloakPod()
 	addRoleCmd := "/opt/jboss/keycloak/bin/kcadm.sh add-roles -r " + targetRealm + " --uusername " + userName + " --cclientid " + clientID + " --rolename " + roleName
-	ctx.Log().Debugf("addRoleCmd: add role %s to the user %s, command = %s", roleName, userName, addRoleCmd)
+	ctx.Log().Debugf("addClientRoleToUser: add role %s to the user %s, command = %s", roleName, userName, addRoleCmd)
```
Comment (kminder) [inline platform-operator/controllers/verrazzano/component/keycloak/keycloak.go:1565]: Not really a fan of adding this type of prefix to user visible messages.  Is this to make it unique from some other message?  If so do that by making the message itself unique.

File: tests/e2e/verify-install/keycloak/keycloak_test.go [hunk 2]
```diff
@@ -500,3 +520,66 @@ func verifyRancherClientURIs(keycloakClient *Client, env string) bool {
...
+	}
+
+	if len(string(out)) == 0 {
+		t.Logs.Error(fmt.Print("Error retrieving client roles JSON from Keycloak, zero length\n"))
+		return false
+	}
+
+	err = json.Unmarshal([]byte(out), &keycloakRoles)
+	if err != nil {
+		t.Logs.Error(fmt.Sprintf("error unmarshalling Keycloak client role json %v\n", err.Error()))
```
Comment (kminder) [inline tests/e2e/verify-install/keycloak/keycloak_test.go:574]: I realize that this is test code but please get in the habit of making logged messages consistent.  For example here it starts with lower case and doesn't capitalize JSON.

File: platform-operator/controllers/verrazzano/component/keycloak/keycloak.go (hunk 1 above)
Comment (bhatpmk) [reply platform-operator/controllers/verrazzano/component/keycloak/keycloak.go:1565]: Yes, I have followed the pattern used in few other functions in keycloak.go. I will update this particular debug message.

File: tests/e2e/verify-install/keycloak/keycloak_test.go (hunk 2 above)
Comment (bhatpmk) [reply tests/e2e/verify-install/keycloak/keycloak_test.go]: Updated the error message.

File: platform-operator/controllers/verrazzano/component/keycloak/keycloak.go (hunk 1 above)
Comment (bhatpmk) [reply platform-operator/controllers/verrazzano/component/keycloak/keycloak.go]: The debug message is updated

**Review by lake-of-dreams — APPROVED** [review]

**Pull request merged**